cfme/fixtures/model_collections.py

```diff
@@ -6,3 +6,11 @@
 def dashboards(appliance):
     from cfme.dashboard import DashboardCollection
     return DashboardCollection(appliance=appliance)
+
+
+@pytest.fixture(scope="module")
```

Comment:
**REQUIRED**: I think all these fixtures should work in the same way if we are going to have them. I don't like that one returns objects and the collection in a list (which is mutable) whereas the dashboards one only returns the collection object.

Note that very soon, this will get a whole lot easier to instantiate due to the changes to the collections. I'm very tempted to say, can we rework this one using new collections but that feels a little unfair. I think, please make it match dashboards above. appliance fixture is available in the test, and it's only currently used once anyway. So just pass the collection, not the objects too. IF we were going to do this like this, I would have mandated that you return it as a set, not a list.